rotkehlchen/accounting/accountant.py

```diff
@@ -606,6 +606,17 @@ def _process_action(
         # When you buy, you buy with the cost_currency and receive the other one
         # When you sell, you sell the amount in non-cost_currency and receive
         # costs in cost_currency
+
+        if trade.rate == 0:
```

Comment:
even though 0 will be coerced to `ZERO` due to the way `FVal` works, `trade.rate` is an FVal. So it's better to compare directly to `ZERO`.